Makefile

```diff
@@ -44,6 +44,10 @@ format:
 	@echo "--> Running go fmt"
 	@go fmt $(PACKAGES)
 
+check-format:
+	@echo "--> Checking format"
+	@! gofmt -l . 2>&1 | read _
+
 test: deps
```

Comment:
why not make it part of the test stage? .. @$(MAKE) check-format ... so it's run every time